src/main/java/seedu/todo/commons/util/TextAreaResizerUtil.java

```diff
@@ -0,0 +1,63 @@
+package seedu.todo.commons.util;
+
+import javafx.beans.value.ChangeListener;
+import javafx.scene.control.TextArea;
+import javafx.scene.text.Text;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+
+/**
+ * Automatically resize a text area object such that the height fits the content of the text area
+ */
+public class TextAreaResizerUtil {
+
+    private Text mockText; //A text object that will imitate the text in the TextArea object
```

Comment:
I'm not entirely sure if "mock" is the best name for this, given that it has a different meaning when used in tests